src/components/ConcurrencyLimitV2ResetModal.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <p-modal v-model:showModal="internalShowModal" class="concurrency-limit-reset-modal" :title="resetTitle">
+    This will reset the active slots count to 0.
+    <template #actions>
+      <slot name="actions">
+        <p-button variant="default" @click="submit">
+          Reset
+        </p-button>
+      </slot>
+    </template>
+  </p-modal>
+</template>
+
+<script lang="ts" setup>
+  import { showToast } from '@prefecthq/prefect-design'
+  import { useSubscription } from '@prefecthq/vue-compositions'
+  import { computed } from 'vue'
+  import { useWorkspaceApi } from '@/compositions'
+  import { localization } from '@/localization'
+  import { ConcurrencyV2Limit } from '@/models/ConcurrencyV2Limit'
+  import { getApiErrorMessage } from '@/utilities/errors'
+
+  const props = defineProps<{
+    showModal: boolean,
+    concurrencyLimit: ConcurrencyV2Limit,
+  }>()
+  const emit = defineEmits<{
+    (event: 'update:showModal', value: boolean): void,
+  }>()
+
+  const internalShowModal = computed({
+    get() {
+      return props.showModal
+    },
+    set(value: boolean) {
+      emit('update:showModal', value)
+    },
+  })
```

Comment:
nit:

You can use [`defineModel`](https://vuejs.org/api/sfc-script-setup.html#definemodel) now for this! 

Simplifies this boilerplate down to something like:
```vue
const props = defineProps<{ concurrencyLimit: ConcurrencyV2Limit }>()
const showModal = defineModel<boolean>("showModal")

// then just use showModal instead of internalShowModal
```

Example - https://github.com/PrefectHQ/prefect-ui-library/blob/ca85777d377dde6c21103f2df4a9b3b6577bfcee/src/components/FlowRunResumeModal.vue#L35-L36

Reply:
Thanks @collincchoy, I'll try that!